barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/verifier.test.cpp

```diff
@@ -137,10 +144,10 @@ template <typename RecursiveFlavor> class RecursiveVerifierTest : public testing
 
         bool broke(false);
         auto check_eq = [&broke](auto& p1, auto& p2) {
+            EXPECT_TRUE(p1.size() == p2.size());
```

Comment:
also could have checked that the circuit sizes of the vks were the same, but this works.